fitbenchmarking/fitbenchmarking
# Failed minimizers report in GUI correction

#### Description of Work

Corrected typos, improved the written expression and removed unnecessary text from the failed minimizers report seen in the GUI if the user specifies minimizers that conflict with the algorithm type option. See new report below, refer to #800 to see previous report.

![image](https://user-images.githubusercontent.com/75368443/120362952-88a89580-c303-11eb-9e44-c46665e1bfa9.png)


#### Testing Instructions
To see this failed minimizer report run an example set of problems with the following specifications in the options file:
1. An "algorithm_type" e.g. "general".
2. A set of minimizers from various software including some that are incompatible with the specified algorithm type.
